charts/values.example.ce.yaml

```diff
@@ -60,3 +60,9 @@ dex:
 minio:
   storeSize: '500Gi'
   storageClass: minio-hdd
+
+telemetry:
+# uuid is the unique identifier of the client where the agent is running.
+# This field is required and will print an error message when that entry is missing.
+# You can generate uuid using command uuidgen on your linux machine
```

Comment:
Please indent the comments like the data, i.e. 2 spaces in this case. Same for the EE example file.